WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -390,7 +393,13 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
     /// - Returns: the requested page.
     ///
     fileprivate func pageAtIndexPath(_ indexPath: IndexPath) -> Page {
-        return _tableViewHandler.page(at: indexPath)
+        if _tableViewHandler.showEditorHomepage {
+            // Since we're adding a fake homepage cell, we need to adjust the index path to match
+            let adjustedIndexPath = IndexPath(row: indexPath.row - 1, section: indexPath.section)
+            return _tableViewHandler.page(at: adjustedIndexPath)
+        } else {
```

Comment:
Small nitpick, but since we're returning from the `if` block, perhaps we can remove the `else` block?

Reply:
Sure! I made both changes in https://github.com/wordpress-mobile/WordPress-iOS/pull/20671/commits/069bd3e55d3fcda4ab7294088baf7ec70c9c29a7.